docs/contracts/NFT Contract/NFT Actions/setprchsreq.a.md

```diff
@@ -22,7 +22,7 @@ The factory manager can specify purchase options for users. Note that currently
 
     -   First, the cost in USD is (factory RAM payment size) \* (RAM price), where
 
-        -   NFT RAM payment size: **1656 bytes**
+        -   NFT RAM payment size: **1679 bytes**
```

Comment:
Also need to update the calculation below:

`(1661B/1024B \* 0.15USD/KB) / (conversion rate) = `0.24331055` **USD**/(conversion rate)`